Model/Reclaim.php

```
@@ -38,4 +69,61 @@ public function stores()
 
         return $hydrated_stores;
     }
+    public function product($filter) {
+        $quote_id = $filter['1'];
+        $item_id = $filter['2'];
+        $quote = $this->quoteFactory->create()->load($quote_id);
+        
+        if (!$quote){
+            return array('error' => 'not found');
+        }
+        $item = $quote->getItemById($item_id);
```

maybe throw an error as well if that item_id wasnt invalid